### DISCUSSION TITLE
Problems importing file exported from notion

### DISCUSSION THREAD

**Discussion by klinux** [outline/outline#5365] [Self Hosting]
I'm trying to import notion exported file, but the import is running for 13 hours, and never ends.

The file that I want to import has 4.1GB, file with 51.35 MB was successful imported.

![Screenshot_763](https://github.com/outline/outline/assets/941051/771af470-d7a9-4a45-9858-812495ea04a9)

No errors in the logs, how can I see if it's processing or not?

Thank you

**Comment by tommoor** [comment]
There is a 2Gb limitation on the size of files that can be opened using NodeJs, the programing language Outline is built in. We should handle this case.
